src/ert/_c_wrappers/enkf/model_config.py

```diff
@@ -222,11 +222,32 @@ def data_root(self):
     def get_time_map(self):
         return self._get_time_map()
 
+    def __repr__(self):
+        return f"<ModelConfig({self.__str__()})>"
+
+    def __str__(self):
+        return (
+            (
+                f"data_root: {self.data_root()},\n"
+                f"num_realizations: {self.num_realizations},\n"
+                f"obs_config_file: {self.obs_config_file},\n"
+                f"ens_path: {self.getEnspath()},\n"
+                f"run_path_format: {self.getRunpathFormat()},\n"
+                f"job_name_format: {self.getJobnameFormat()},\n"
+                f"run_path: {self.getRunpathAsString()},\n"
+                f"max_internal_submit: {self.get_max_internal_submit()},\n"
+                f"gen_kw_export_name: {self.getGenKWExportName()},\n"
+                f"time_map: {self.get_time_map()},\n"
+            )
+            if self._address()
```

Comment:
`if self._address()` then what? Is something missing here?

Reply:
it's perhaps an awkwardly formatted ternary thingy - 

`A if B else C` but written as

```
(AA
AAAA
AA
AAA
AAA
A)
if B
else C
```

Reply:
ah.... i didn't notice at first :facepalm:

Reply:
no worries, it is strangely formatted : )